cmake/haros_catkin-extras.cmake.em

```diff
@@ -15,10 +15,29 @@ macro(_haros_create_targets)
 endmacro()
 
 function(haros_report)
-  set(HAROS_REPORT_LOCATION "${CMAKE_CURRENT_BINARY_DIR}/test_results/haros_report")
+  cmake_parse_arguments(
+    HAROS
+    ""
+    "REPORT_LOCATION;CONFIG_PATH;HOME_PATH"
+    ""
+    ${ARGN} )
+  if(NOT HAROS_REPORT_LOCATION)
+    set(HAROS_REPORT_LOCATION "${CMAKE_CURRENT_BINARY_DIR}/test_results/haros_report")
+  else()
+    set(HAROS_REPORT_LOCATION "${HAROS_REPORT_LOCATION}/${CMAKE_PROJECT_NAME}")
```

Comment:
This is slightly confusing: if the user specifies a `HAROS_REPORT_LOCATION`, we're still appending the project name here. I would not have expected that as a user.

It's fine if we want to do this, but could we add a comment to the function that mentions this (similar to something like [this](https://github.com/ros/catkin/blob/461cd3c217f952f908a3cba56927660764b9ae62/cmake/catkin_package.cmake))?

Reply:
You're right -  I have updated a comment to the `haros_report` function which should clarify this (among the other arguments).
However I'd be fine to remove this feature, too as it's probably unnecessary and maybe not the right place. I guess HAROS itself should probably do this.

Reply:
> However I'd be fine to remove this feature, too as it's probably unnecessary and maybe not the right place. I guess HAROS itself should probably do this.

I'm not sure I understand this?

Reply:
The idea was to let `haros` decide to use a proper subfolder for each package's report of interest instead of doing this in `haros_catkin`.

But I guess appending the `CMAKE_PROJECT_NAME` has the benefit that `haros` will be create its own database for each package of interest (and not load an existing one as mentioned [here](https://github.com/git-afsantos/haros/tree/d16df675109f98592a73439a74f06e8cb7d00c25#haros-analyse--d-data_dir))